includes/HttpFunctions.php

```diff
@@ -431,7 +432,17 @@ public function getHeaderList() {
 		// SUS-5499: Use internal host name for MW->MW requests when running on Kubernetes
 		global $wgKubernetesNamespace;
 		if ( !empty( $wgKubernetesNamespace ) && Http::isLocalURL( $this->url ) ) {
-			$list[] = sprintf( 'X-Original-Host: %s', $this->parsedUrl['host'] );
+			$originalHost = $this->parsedUrl['host'];
```

Comment:
cannot we simply go with this instead of duplicating the code that replaces part of the host?
`
$list[] = sprintf( 'X-Original-Host: %s', wfNormalizeHost( $this->parsedUrl['host'] ) );
`

Reply:
Implemented in 6454b28